04_shortlinker_api/src/controllers/url.ts

```diff
@@ -0,0 +1,5 @@
+import {Request, Response, NextFunction} from 'express';
+
+export const shortUrl = async (req: Request, res: Response, next: NextFunction) => {
+    //TODO: Implement
+}
```

Comment:
Do you really need to commit this?

Reply:
Don't worry, this is still a WIP version. I'm working on it.